CMakeLists.txt

```diff
@@ -208,3 +208,33 @@ configure_file("${CMAKE_CURRENT_SOURCE_DIR}/cmake/redis++.pc.in"
 
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/cmake/redis++.pc"
         DESTINATION "lib/pkgconfig")
+
+option(REDIS_PLUS_PLUS_BUILD_DEBIAN "Build Debian package" OFF)
+
+if (REDIS_PLUS_PLUS_BUILD_DEBIAN)
```

Comment:
Since some of the cpack related stuff depends on cmake 3.6, e.g. CPACK_DEBIAN_PACKAGE_GENERATE_SHLIBS, we should only enable cpack related stuff if the cmake version is larger than 3.6:

```
if(${CMAKE_VERSION} VERSION_GREATER 3.6)
```